axelrod/fingerprint.py

```diff
@@ -242,32 +92,37 @@ def fingerprint(self, turns=50, repetitions=10, step=0.01, processes=None,
         repetitions : integer, optional
             The number of times the round robin should be repeated
         step : float, optional
+            0.0 < step <= 1.0
             The separation between each Point. Smaller steps will
             produce more Points that will be closer together.
         processes : integer, optional
             The number of processes to be used for parallel processing
+        filename :
+            Location of csv of tournament.interactions_dict
+        in_memory :
+            True=tournament.interactions_dict is stored only in memory
+            False=tournament.interactions_dict is stored only on file
         progress_bar : bool
             Whether or not to create a progress bar which will be updated
 
         Returns
         ----------
-        self.data : dictionary
+        self.data : dictionary(Point, float)
             A dictionary where the keys are coordinates of the form (x, y) and
             the values are the mean score for the corresponding interactions.
         """
 
-        if on_windows and (filename is None):  # pragma: no cover
+        if axl.on_windows and (filename is None):  # pragma: no cover
```

Comment:
so, i just want to confirm.  keep this and the relevant tests to go with it, or ditch it?  (all the other changes will be gone in a moment.)